lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -134,34 +149,36 @@ def update_positions_if_necessary
         end
 
         def update_positions
-          tn = ActiveRecord::Base.connection.quote_table_name acts_as_list_class.table_name
-          pk = ActiveRecord::Base.connection.quote_column_name acts_as_list_class.primary_key
-          up = ActiveRecord::Base.connection.quote_table_name "updated_positions"
+          unless ActsAsList.skip_cb
```

Comment:
Might be better to do this [on the callback declaration itself](https://github.com/aha-app/acts_as_list/pull/3/commits/a63b2af4f77f12502fcb1f5e09ba6310b916ee6b#diff-708784f0c89e5cdbdd54e18d2b2243d1R132) via [callback conditionals](http://guides.rubyonrails.org/active_record_callbacks.html#using-if-and-unless-with-a-symbol) so that anything that overrides/extends this method doesn't lose the callback skipping.